Utils/src/FastSimWeightPR31285To36122.cc

```diff
@@ -0,0 +1,246 @@
+// -*- C++ -*-
+// Package:    TreeMaker
+// Class:      FastSimWeightPR31285To36122
+// Authors:   Sam Bein 
+//         Created:  Wed March 7, 2014
+//         Modified: Thurs March 3, 2021
+
+#include <memory>
+#include <iostream>
+#include <unordered_set>
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/global/EDProducer.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+#include "CommonTools/UtilAlgos/interface/TFileService.h"
+#include <DataFormats/HepMCCandidate/interface/GenParticle.h>
+#include "DataFormats/JetReco/interface/GenJet.h"
+#include "DataFormats/PatCandidates/interface/Jet.h"
+#include "TLorentzVector.h"
+
+#include <vector>
+
+class FastSimWeightPR31285To36122 : public edm::global::EDProducer<> {
+
+public:
+    explicit FastSimWeightPR31285To36122(const edm::ParameterSet&);
+    ~FastSimWeightPR31285To36122() override;
```

Comment:
unnecessary, remove

Reply:
Are you referring to the 4 lines indicated? It seems this results in a compilation error, also when removing the lines in the following comment as well.

Reply:
(if I remove just line 30, then it's fine)

Reply:
Then just set it inline: `~FastSimWeightPR31285To36122() override {}`